### PR TITLE
tests: build with DDEBUG=1.

### PR DESCRIPTION
**Pull request by zhuizhuhaomeng**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

**Review by cppcoffee — DISMISSED** [review]

File: util/build-with-dd.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            --with-http_image_filter_module \
+            --without-mail_smtp_module \
+            --with-stream \
+            --with-stream_ssl_module \
+            --without-http_upstream_ip_hash_module \
+            --without-http_memcached_module \
+            --without-http_auth_basic_module \
+            --without-http_userid_module \
+            --with-http_auth_request_module \
+                --add-module=$root/../echo-nginx-module \
```
Comment (cppcoffee) [inline util/build-with-dd.sh:37]:
```suggestion
            --add-module=$root/../echo-nginx-module \
```

Comment (cppcoffee) [reply util/build-with-dd.sh:37]: Extra spaces, we can delete them.

File: util/build-with-dd.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                --add-module=$root/../stream-lua-nginx-module \
+                --add-module=$root/t/data/fake-module \
+                $add_fake_shm_module \
+                --add-module=$root/t/data/fake-delayed-load-module \
+                --with-http_gunzip_module \
+                --with-http_dav_module \
+          --with-select_module \
+          --with-poll_module \
+                $opts \
+                --with-debug
```
Comment (cppcoffee) [inline util/build-with-dd.sh:56]:
```suggestion
            --add-module=$root/../echo-nginx-module \
            --add-module=$root/../memc-nginx-module \
            --add-module=$root/../srcache-nginx-module \
            --add-module=$root \
            --add-module=$root/../lua-upstream-nginx-module \
            --add-module=$root/../headers-more-nginx-module \
            --add-module=$root/../drizzle-nginx-module \
            --add-module=$root/../rds-json-nginx-module \
            --add-module=$root/../coolkit-nginx-module \
            --add-module=$root/../redis2-nginx-module \
            --add-module=$root/../stream-lua-nginx-module \
            --add-module=$root/t/data/fake-module \
            $add_fake_shm_module \
            --add-module=$root/t/data/fake-delayed-load-module \
            --with-http_gunzip_module \
            --with-http_dav_module \
            --with-select_module \
            --with-poll_module \
            $opts \
            --with-debug
```

Comment (cppcoffee) [reply util/build-with-dd.sh:56]: Extra spaces, we can delete them.

**Review by spacewander — APPROVED** [review]

**Review by cppcoffee — APPROVED** [review]

**Pull request merged**